components/tekton/pipelineruns/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Not sure if we need this file here, since this is a catalog it should contain the kustomization file needed for creating a pipeline/pipelinerun. Also, maybe we won't need to use all the pipelines in the same cluster.
I think we will use these kustomizations in the hub

Reply:
Went through similar thoughts last week. In my view, tasks and pipelines templates should be available. 

The kustomization file in the hub is the one that creates meaningful pipelineRuns CRs depending on cluster and testing needs with the proper parameters. 
Comment accepted.

> Also, maybe we won't need to use all the pipelines in the same cluster.
Agree. That we will control using kustomization file in the hub.